plugins/CoreHome/vue/src/ShowHelpLink/ShowHelpLink.vue

```diff
@@ -0,0 +1,47 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+<a
+  class="item-help-icon"
+  tabindex="5"
+  href="javascript:"
+  v-if="message"
+  @click="showHelp"
+>
+<span class="icon-help" />
+</a>
+</template>
+<script lang="ts">
+
+import { defineComponent } from 'vue';
+import { NotificationsStore } from '../Notification';
+
+const REPORTING_HELP_NOTIFICATION_ID = 'reportingmenu-help';
+
+export default defineComponent({
+  props: {
+    message: {
+      type: String,
+      default: null,
+    },
+  },
+  methods: {
+    showHelp() {
+      NotificationsStore.show({
+        context: 'info',
+        id: REPORTING_HELP_NOTIFICATION_ID,
+        type: 'help',
+        noclear: true,
+        class: 'help-notification',
+        message: this.message,
+        placeat: '#notificationContainer',
+        prepend: true,
+      });
+    },
```

Comment:
This shows the help notification well and clicking on a different help icon replaces the help text with that help text. However, there's one thing that this isn't doing that `plugins/CoreHome/vue/src/ReportingMenu/ReportingMenu.vue` does. When I click the same help icon twice, this code doesn't hide the purple notification. For example, from the Matomo dashboard, I select the Goals menu item and then click the help icon next to the Overview submenu to view the purple notification containing the help text. If I click that same help icon a second time, the purple notification is hidden.

Reply:
Thank you @peterhashair . It now hides when I click the icon a second time.

Every once in a while there's some odd behaviour when I click on the help icon next to one submenu and then the icon next to another submenu. In some cases it hides the notification instead of replacing it. Sometimes I have to click the help icon multiple times. Although, it's enough of a fringe case that it's probably fine to let through.